src/components/AddItemForm.js

```diff
@@ -69,6 +84,21 @@ class AddItemForm extends React.Component {
         </label>
         <br />
         <input className="submit-btn" type="submit" value="Submit" />
+        <FirestoreCollection
+          path="items"
+          filter={['name_normalized', '==', normalizedInput]}
+          render={({ data }) => {
+            let message = <div></div>;
```

Comment:
Not a blocker. I wonder if this should be `null` or an empty `p` tag . . . ? Just because it's kind of confusing that `message` is initialized as an empty `div` then is changed to a `p` when an alert is made. (My bad.)

Reply:
this was fixed with the suggestion to `short circuit pattern for a render this or nothing`. Thanks, Scott I understand what that meant now!